docker/local-universe/README.md

```diff
@@ -190,3 +190,13 @@ For example, if you want dev-universe, then add `--server_url http://dev-univers
   ]
 }
 ```
+
+## Making changes...
+
+###  ...to the `universe-static` image
+
+1. Update `Dockerfile.static` with the changes
+1. Bump the default value of `static_version` in `Makefile`
+1. Create a pull request with the above changes
+1. Once it is merged, run the [Publish universe-static image](https://teamcity.mesosphere.io/viewType.html?buildTypeId=Oss_Universe_2_PublishUniverseStaticImage&branch_Oss_Universe_2=%3Cdefault%3E&tab=buildTypeStatusDiv) build, providing the new version number when prompted
```

Comment:
I haven't updated the build steps yet, because one of the steps actually gets more verbose with this change! It's because the `Makefile` doesn't build the image with the `mesosphere/` prefix, so we have to tag it separately. Here's the current build step:

```bash
#!/bin/bash
set -o errexit -o nounset -o pipefail

docker build -t "mesosphere/universe-static:%docker.image.version%" -f Dockerfile.static .

exit
```

And here's what we'd have to update it to:
```bash
#!/bin/bash
set -o errexit -o nounset -o pipefail

make static_version=%docker.image.version% static-build
docker tag universe-static:%docker.image.version% "mesosphere/universe-static:%docker.image.version%"

exit
```

@jsancio any suggestions?